scripts/JsonRpc.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+
+class JsonRpc(requests.Session):
+    rpc_path = '/json-rpc'
+
+    def __init__(self, address):
```

Comment:
please rename `address` to `host` and have `self.address = host + self.rpc_path`